FXyz-Importers/src/main/java/org/fxyz3d/importers/maya/MayaImporter.java

```diff
@@ -75,7 +78,7 @@ public MayaGroup getRoot() {
     // MayaImporter.getTimeline() returns a JavaFX timeline
     // (javafx.animation.Timeline)
     //=========================================================================
-    @Override
+
     public Timeline getTimeline() {
```

Comment:
since now we have a model, I'd remove these two methods `getRoot()` and `getTimeline()`.